enhancements/etcd/automated-backups.md

```diff
@@ -65,101 +67,147 @@ node.
 
 ### Workflow Description
 
-To enable automated backups a new singleton CRD can be used to specify the backup schedule and other configuration as well as monitor the status of the backup operations. 
+#### One time backups
+
+To enable one-time backup requests via an API, a new CRD, `EtcdBackupRequest` (open to better name suggestions), will be used to trigger one-time backup requests.
+
+A new controller in the cluster-etcd-operator, `EtcdBackupRequestController`, will reconcile backup requests as follows:
+- Watch for new `EtcdBackupRequest` CRs as created by an admin
+- Create a backup Job configured for the backup request spec
+- Track the backup progress, failure or success on the `EtcdBackupRequest` status
```

Comment:
I assume you will also need to track:
* file path
* node name

so we can later sort out the retention, correct?